velox/exec/RowContainer.h

```diff
@@ -743,6 +808,11 @@ class RowContainer {
     return types_;
   }
 
+  /// Returns the aggregated column stats of the column corresponds to requested
+  /// 'columnIndex'. nullopt will be returned if the column is not tracked
```

Comment:
```
The function returns ...
```

Reply:
Subject of method description is assumed to be the method itself. This aligns with other documentations as well.